apps/arena/lib/arena/game_updater.ex

```diff
@@ -1138,7 +1138,7 @@ defmodule Arena.GameUpdater do
     entities_to_collide_with =
       Player.alive_players(players)
       |> Map.merge(Obstacle.get_collisionable_obstacles_for_projectiles(obstacles))
-      |> Map.merge(crates)
+      |> Map.merge(crates |> Map.filter(fn {_id, crate} -> crate.aditional_info.status != :DESTROYED end))
```

Comment:
Could we use `Crate.alive?` here?

Reply:
No, but we can use our brand new Crate.alive_crates/1 :altoken: